README.md

```diff
@@ -26,7 +26,7 @@
     </a>
 </p>
 
-Haystack is an end-to-end framework that enables you to build powerful and production-ready pipelines for different search use cases.
+[Haystack](https://haystack.deepset.ai) is an end-to-end framework that enables you to build powerful and production-ready pipelines for different search use cases.
```

Comment:
We shouldn't be saying our products or things related to our products "enable the users" to do something (apparently that sends a wrong message) so how about: "..framework for you to build powerful and..."

Reply:
can we make this PR only about updating the URLs please? We can change the messaging etc as a separate exercise and we can go through the whole readme

Reply:
ah alright, sure, just can't help myself.. ;)